src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -105,6 +103,8 @@ MFEMSidreDataCollection::MFEMSidreDataCollection(const std::string& collection_n
 
   m_named_bufs_grp = domain_grp->createGroup("named_buffers");
 
+  this->own_data = own_mesh_data;
```

Comment:
@jamiebramwell -- this change might affect serac -- if you call this constructor.

If so, you'd need to add `dc->SetOwnData(false)` after calling the `MFEMSidreDataCollection` constructor. There are numerous examples in the sidre unit tests and examples.